### PR TITLE
Updating redirect links in the WasmEdge docs 

### LINKED ISSUE #2716: [Docs] update redirect links in the WasmEdge docs folder
## Description

WasmEdge split the docs from this main repository to a brand new one, [WasmEdge/docs](https://github.com/WasmEdge/docs). Since some files are moved to different places, the original link may not work.
This issue is for tracking these unresolved/non-updated links to up-to-date.

### Current State

For example, links in [this page](https://wasmedge.org/book/en/write_wasm/rust/wasinn.html) are not available because there are [several refactoring PRs](https://github.com/WasmEdge/docs/pulls?q=is%3Apr+is%3Aclosed+author%3Aq82419) which changed the paths.

### Expected

All links in the [original WasmEdge docs](https://wasmedge.org/book/en/) should work with [the WasmEdge docs](https://wasmedge.org/docs).

If you find any link is wrong, please raise PRs to fix them.

### PR DESCRIPTION
**Pull request by Mash707**
resolves #2716

### CODE REVIEW

**Comment by juntao** [comment]
Hello, I am a [code review bot](https://github.com/flows-network/github-pr-summary/) on [flows.network](https://flows.network/).

It could take a few minutes for me to analyze this PR. Relax, grab a cup of coffee and check back later. Thanks!

**Comment by Mash707** [comment]
Hii @hydai . I am having some issue regarding two links 
1. https://wasmedge.org/book/en/features/proprietary_extend ,I am not able to find this in the new docs.
2. https://wasmedge.org/book/en/write_wasm/rust/tensorflow , regarding this I found this on the news docs but don't know if this is the correct one https://wasmedge.org/docs/develop/rust/tensorflow ?

**Comment by hydai** [comment]
Hi @alabulei1 
Could you please help review this PR?

File: docs/book/en/src/features/proposals.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # Supported WASM And WASI Proposals
 
-> *This part has moved to <https://wasmedge.org/docs/develop/wasmedge/extensions/proposals>. Please use our new docs.*
+> *This part has moved to <https://wasmedge.org/docs/start/wasmedge/extensions/proposals>. Please use our new docs.*
 
 ## WebAssembly proposals
```
Comment (alabulei1) [inline docs/book/en/src/features/proposals.md:5]: Can you please also help me remove the content?

File: docs/book/en/src/quick_start/install.md [hunk 2]
```diff
@@ -1,6 +1,6 @@
 # WasmEdge Installation And Uninstallation
 
-> *This part has moved to <https://wasmedge.org/docs/develop/build-and-run/install>. Please use our new docs.*
+> *This part has moved to <https://wasmedge.org/docs/start/install>. Please use our new docs.*
 
 ## Quick Install
```
Comment (alabulei1) [inline docs/book/en/src/quick_start/install.md:5]: Can you please also help me remove the content?

File: docs/book/en/src/quick_start/run_cli.md [hunk 3]
```diff
@@ -1,6 +1,6 @@
 # Running WASM with WasmEdge CLI
 
-> *This part has moved to <https://wasmedge.org/docs/develop/build-and-run/cli>. Please use our new docs.*
+> *This part has moved to <https://wasmedge.org/docs/start/build-and-run/cli>. Please use our new docs.*
 
 After [installing WasmEdge](install.md) or starting the [WasmEdge app dev Docker container](use_docker.md), there are several ways to run WebAssembly programs.
```
Comment (alabulei1) [inline docs/book/en/src/quick_start/run_cli.md:5]: Can you please also help me remove the content?

File: docs/book/en/src/quick_start/run_in_aot_mode.md [hunk 4]
```diff
@@ -1,6 +1,6 @@
 # Execution in AOT Mode
 
-> *This part has moved to <https://wasmedge.org/docs/develop/build-and-run/aot>. Please use our new docs.*
+> *This part has moved to <https://wasmedge.org/docs/start/build-and-run/aot>. Please use our new docs.*
 
 The `wasmedge` command line tool will execute the original WASM files in interpreter mode. For the much better performance, we recommend users to compile the WASM with the `wasmedge compile` AOT compiler to execute the WASM in AOT mode. There are 2 output formats of the AOT compiler:
```
Comment (alabulei1) [inline docs/book/en/src/quick_start/run_in_aot_mode.md:5]: Can you please also help me remove the content?

**Review by alabulei1 — APPROVED** [review]

**Comment by Mash707** [comment]
Hii @alabulei1 Do you mean to remove the whole content and just leave the redirect links?

**Comment by alabulei1** [comment]
Yes. That's what I mean!

**Comment by Mash707** [comment]
> Hii @hydai . I am having some issue regarding two links 
> 
> 1. https://wasmedge.org/book/en/features/proprietary_extend ,I am not able to find this in the new docs.
> 
> 2. https://wasmedge.org/book/en/write_wasm/rust/tensorflow , regarding this I found this on the news docs but don't know if this is the correct one https://wasmedge.org/docs/develop/rust/tensorflow ? 

@alabulei1 Could please have a look at this too.

**Comment by Mash707** [comment]
Hii @alabulei1 @hydai , I have made the necessary changes. Please have a look if something else needs to be done. 
Thank you.

**Review by alabulei1 — APPROVED** [review]

**Comment by alabulei1** [comment]
LGTM!

**Pull request merged**